src/auth/tenant.ts

```diff
@@ -21,7 +21,7 @@ import { AuthClientErrorCode, FirebaseAuthError } from '../utils/error';
 import {
   EmailSignInConfig, EmailSignInConfigServerRequest, MultiFactorAuthServerConfig,
   MultiFactorConfig, validateTestPhoneNumbers, EmailSignInProviderConfig,
-  MultiFactorAuthConfig,
+  MultiFactorAuthConfig, RecaptchaConfigAuth, RecaptchaConfig
```

Comment:
Change to *AuthConfig to match MultiFactorAuthConfig?

Reply:
Done.